### ISSUE TITLE
DMTet without finetuning

### ISSUE THREAD

**Issue by ug-kim** [ashawkey/stable-dreamfusion#226]
Hello, I have some questions about DMTet fine-tuning.

Can I train 3D with only DMTet without NeRF pre-training?

I try to this script but failed.

`python main.py -O --image data/firekeeper_rgba.png --workspace work_dirs/trial_image_dmtet --dmtet`
RuntimeError: instance mode - pos must have shape [>0, >0, 4]

I have one more question. 

When I finetune with DMTet, the output representation is DMTet and not NeRF.
In other words, is the initial NeRF pre-train not updated, and only DMTet fine-tuned?

Thanks for sharing your amazing work.

**Comment by ashawkey** [comment]
@ug-kim Hi, currently we don't support directly start from DMTet, maybe you could check Fantasia3D for that.
Not exactly, the NeRF's color grid is loaded into DMTet and get finetuned.

**Comment by ug-kim** [comment]
Thank you for your fast answer
